### ISSUE TITLE
Package name should follow the parent package

### ISSUE THREAD

**Issue by alaminfirdows** [LaravelDaily/laravel-permission-ui#1]
This is going to be the most popular extension for managing the `spatie/laravel-permission` with a great user interface. I'm super excited to use this package for my future project.

I am going through this package and found that spatie publish their package as `spatie/laravel-permission`. As this package is extending spatie's laravel permission package, you should follow the parent and set the package name as `laraveldaily/laravel-permission-ui`. (singular form)

It will help users to relate to the parent package and manage easily.

```diff
- laraveldaily/laravel-permissions-ui
+ laraveldaily/laravel-permission-ui
```

**Comment by atiq-ur** [comment]
I appreciate it.

**Comment by PovilasKorop** [comment]
@alaminfirdows I agree, good suggestion, will rename.

**Comment by alaminfirdows** [comment]
@PovilasKorop I’m waiting to submit some other PRs, depending on this issue.

**Comment by PovilasKorop** [comment]
@alaminfirdows thanks, for now hold on with PRs, we got a lot of feedback so will fundamentally restructure some things for v0.2 in upcoming week(s) and then will accept PRs.

**Comment by PovilasKorop** [comment]
@alaminfirdows ok renamed the package and the config, released `v0.1.2`. Will move on to other issues, one by one.
